### PR TITLE
Update instructions on using Postgres Database in Dockerfile

### PR DESCRIPTION
**Pull request by Christopher-Chianelli**
Replaces #652 

### Referenced pull requests

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>full downstream build</b> please add comment: <b>Jenkins run fdb</b>
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,21 +15,26 @@
...
+# Build an image (that uses PostgreSQL database):
+# docker build -t optaweb/employee-rostering --build-arg QUARKUS_PROFILE=postgres --ulimit nofile=98304:98304 .
+#
+# Run the image (if built for in-memory H2 database):
 # docker run -p 8080:8080 --rm -it optaweb/employee-rostering
 #
-# Run the image with production profile (using PostgreSQL database):
-# docker run -p 8080:8080 --rm -it -e QUARKUS_PROFILE=production optaweb/employee-rostering
+# Run the image (if build for PostgreSQL database):
+# docker run -p 8080:8080 --rm -it -e DATABASE_NAME=$NAME -e DATABASE_USER=$USER -e DATABASE_PASSWORD=$PASSWORD optaweb/employee-rostering
```
Comment (triceo) [inline Dockerfile:28]: Not sure where this information comes from. Is the user expected to run an external database somewhere?
The instructions don't really say.

Comment (otherwish) [reply Dockerfile:28]: As far as I can tell, the postgres profile is explicitly for an external database. Passing those variables with the added build arguments did work for my build.

Comment (Christopher-Chianelli) [reply Dockerfile:28]: It for an external database deployed on the local machine (either from a different docker image, setup on local machine, etc.). If a cloud provider/outside database is used w/o URL remapping for `postgres`, the environment variable `DATABASE_HOST` would also need to be set to the database location.

Comment (triceo) [reply Dockerfile:28]: @Christopher-Chianelli I'd say that deliberately in the comments. `DATABASE_HOST` is not mentioned there at all, for example. If we're using the comments as a howto, let's describe the use cases.

**Comment by triceo** [comment]
`optaweb-employee-rostering` project is no longer maintained, closing this PR.

**Pull request closed without merging**